README.md

```diff
@@ -52,5 +52,94 @@ More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
 ![Ten Pin Score Example](images/example_ten_pin_scoring.png)
 
 ## My Solution
+A Makers Week 5 solo weekend challenge. 
+
+* **Languages used**: Ruby
+* **Testing frameworks**: RSpec, Capybara
+* **Linting**: Rubocop
 
 ### Domain Model
+| Class | Game |
```

Comment:
Wow, proper planning! - I'm impressed with how you used Domain modelling technique here. well done!